### PR TITLE
Add tron owners to mufex

### PR DESCRIPTION
**Pull request by zhangtengfei91**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/mufex exports TVL: 
        
 
 
    polygon                   25.44 k
    arbitrum                  6.03 k
    bsc                       13.40 k
    ethereum                  5.00 k
    mantle                    27.90 k
    tron                      7.20 k
    
    total                    84.97 k

**Comment by g1nt0ki** [comment]
hi @zhangtengfei91 thanks for the PR, I have not merged it yet because, these appear to be EOA instead of smart contracts?

**Comment by llamatester** [comment]
The adapter at projects/mufex exports TVL: 
        
 
 
    bsc                       13.43 k
    polygon                   24.64 k
    arbitrum                  6.04 k
    mantle                    27.94 k
    tron                      6.29 k
    
    total                    78.34 k

**Comment by zhangtengfei91** [comment]
> hi @zhangtengfei91 thanks for the PR, I have not merged it yet because, these appear to be EOA instead of smart contracts?

hi @g1nt0ki I have updated the code, please help to check, thank you.

**Comment by llamatester** [comment]
The adapter at projects/mufex exports TVL: 
        
 
 
    polygon                   24.59 k
    arbitrum                  6.03 k
    bsc                       13.40 k
    mantle                    27.89 k
    tron                      6.28 k
    
    total                    78.19 k

**Comment by llamatester** [comment]
The adapter at projects/mufex exports TVL: 
        
 
 
    bsc                       13.40 k
    polygon                   24.59 k
    mantle                    27.89 k
    tron                      6.28 k
    arbitrum                  6.03 k
    
    total                    78.19 k

File: projects/mufex/index.js [hunk 1]
```diff
@@ -6,9 +6,14 @@ const owners = [
   '0x16BEDB2Ab2aEf9023ff2cbF0C78135cA120c03C6',
 ]
 
+const tronOwners = [
+  'TUuRXVc9r5NYVq9fQL5gHTMm85vr3nj9oo',
```
Comment (g1nt0ki) [inline projects/mufex/index.js:10]: this is still not a smart contract?

**Comment by g1nt0ki** [comment]
closed as there was no answer

**Pull request closed without merging**